### ISSUE TITLE
Game hangs on first attempt to run (Linux)

### ISSUE THREAD

**Issue by ajgringo619** [triplea-game/triplea#8823]
### How can the problem be recreated?
After installing, either in /opt or $HOME, the game refuses to start.

### Do you have any ideas for an expected fix?
My guess is a java version problem.

### Attach a Save Game
Never get this far.

### If playing a prerelease, which version are you using?

Game Version:

### If playing a prerelease, does this happen on the latest release?

- [] yes
- [] no

### Is there anything else we should know?
Here's the error log:
```
$ cat error.log 
java.lang.ClassNotFoundException: org.triplea.game.client.HeadedGameRunner
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:602)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at com.exe4j.runtime.LauncherEngine.launch(LauncherEngine.java:80)
	at com.install4j.runtime.launcher.UnixLauncher.start(UnixLauncher.java:66)
	at install4j.org.triplea.game.client.HeadedGameRunner.main(Unknown Source)
Uncaught error fetching image:
java.lang.NullPointerException
	at java.desktop/sun.awt.image.URLImageSource.getConnection(URLImageSource.java:101)
	at java.desktop/sun.awt.image.URLImageSource.getDecoder(URLImageSource.java:111)
	at java.desktop/sun.awt.image.InputStreamImageSource.doFetch(InputStreamImageSource.java:263)
	at java.desktop/sun.awt.image.ImageFetcher.fetchloop(ImageFetcher.java:212)
	at java.desktop/sun.awt.image.ImageFetcher.run(ImageFetcher.java:176)
```

Here are my system specs:
Linux Mint 20.1
openjdk 14.0.2 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-Ubuntu-120.04)
OpenJDK 64-Bit Server VM (build 14.0.2+12-Ubuntu-120.04, mixed mode, sharing)
32 GB RAM, GeForce GTX 1070 w/8 GB RAM (Nvidia v460.32.03)

**Comment by DanVanAtta** [comment]
How are you installing the game?

**Comment by ajgringo619** [comment]
After making the script executable, I simply do `./TripleA_2.5.22294_unix.sh`. Should I be running this as root?

**Comment by DanVanAtta** [comment]
As any standard user, not as root.
Do you have any additional firewall or other security apps installed?

**Comment by ajgringo619** [comment]
I use `ufw`, standard with Mint. Let me try again with it disabled.

**Comment by DanVanAtta** [comment]
Also, how exactly are you launching the game?

**Comment by DanVanAtta** [comment]
One more question, is the crash just after install and then you can launch, or are you unable to launch the game entirely?

**Comment by ajgringo619** [comment]
It crashes whenever I actually start the game. I've tried (2) different ways: 1) using the option to start TripleA right after install, or 2) running `./TripleA` from the install directory. I've tried using both JRE 11 and 14, both give the same errors.

**Comment by ajgringo619** [comment]
It's not my router firewall either, as my Windows 10 (dual-boot) runs the game without a hitch.

**Comment by DanVanAtta** [comment]
Do you have any luck if you run:
```
java -jar bin/triplea-game-headed-2.5.22294.jar
```

**Comment by ajgringo619** [comment]
I get this, whether I use v11 or v14 of java:
`Error: Invalid or corrupt jarfile bin/triplea-game-headed-2.5.22294.jar`

**Comment by ajgringo619** [comment]
Here's more info I found from the installation log:
`[ERROR] com.install4j.runtime.beans.actions.InstallFilesAction [ID 9]: Cannot not unpack /home/me/TripleA/bin/i4j803212607638661800.tmp on Java 14`

**Comment by ajgringo619** [comment]
I got it working. Switching my java version to v11 allowed the installer to unpack the aforementioned jar file. I will close this, but if anyone else is using java v14 it may not work.

**Comment by seanmkauffman** [comment]
I do not seem to be able to get it to work with any of the JDKs I have access to.  Is there anyone that is currently able to run TripleA on Linux with a modern JDK version?  The workaround in this thread says to downgrade to v11, which is not really an option any more.
